src/gordon_janitor_gcp/plugins/reconciler.py

```diff
@@ -216,16 +216,19 @@ def __init__(self, config, dns_client, rrset_channel=None,
             }
             # TODO (lynn): add metrics.incr call here once aioshumway is
             #              released
-            logging.debug('Creating the following change message: {msg}')
+            logging.debug(f'Creating the following change message: {msg}')
             await self.changes_channel.put(msg)
 
         logging.info(f'Created {len(desired_rrsets)} change messages.')
 
+    def _zone_to_managed_zone(self, zone):
+        return zone.replace('.', '-')[:-1]
```

Comment:
Are you thinking of Gordon-core? Cause here there's no `managed_zone` config and the managed zones are created on the fly from instance information I believe.

Reply:
We talked about this IRL and I was wrong - this change should not belong in the public janitor code because this zone->managedZone mapping is not generic enough. Thanks @ocsw !